packages/lambda/lib/aws-config.js

```diff
@@ -1,6 +1,12 @@
 'use strict';
 
 const semver = require('semver');
+const {
+  stripLeadingSlash,
+  stripTrailingSlash,
+} = require('@untool/express').uri;
+
+const trimSlashes = input => stripLeadingSlash(stripTrailingSlash(input));
```

Comment:
Why not put `trimSlashes` into a file inside /lib and use it here and in `lambda.js`?

Reply:
I created a PR in untool: https://github.com/untool/untool/pull/125